File: 1035.py
x = input().split()
a, b, c, d = x
a = int(a)
b = int(b)
c = int(c)
d = int(d)
if b > c and d > a and (c + d) > (a + b) and c > 0 and d > 0 and a % 2 == 0:
    print('Valores aceitos')
else:
    print('Valores nao aceitos')

File: C#/1035.cs
using System; 
using System.Linq;

class URI {

    static void Main(string[] args) { 

        int[] x = Console.ReadLine().Split().Select(int.Parse).ToArray();
        int a=x[0];
        int b=x[1];
        int c=x[2];
        int d=x[3];

        if(b>c &&d>a &&c+d>a+b &&c>=0 &&d>=0 &&a%2==0)
        {
            Console.WriteLine("Valores aceitos");
        }
        else{Console.WriteLine("Valores nao aceitos");}

    }

}